htdocs/compta/facture/list.php

```diff
@@ -145,6 +145,10 @@
 $search_btn = GETPOST('button_search', 'alpha');
 $search_remove_btn = GETPOST('button_removefilter', 'alpha');
 
+$search_status = GETPOST('search_status', 'alpha');
+$optioncss = GETPOST('optioncss', 'alpha');
+$object_statut = GETPOST('search_statut', 'alpha');
```

Comment:
Why adding search_statut ? Seems a duplicate of search_status.
Filter on search_status seems already working.